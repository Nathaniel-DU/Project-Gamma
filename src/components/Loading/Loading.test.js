import React from 'react';
import { shallow } from 'enzyme';
import Loading from './index';

describe('Loading', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Loading debug />);
        expect(component).toMatchSnapshot();
    })
})