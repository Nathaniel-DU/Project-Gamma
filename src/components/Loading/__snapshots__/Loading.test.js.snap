// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly in "debug" mode 1`] = `
<div
  className="spinner"
>
  <img
    alt="Loading"
    src="loading.svg"
  />
</div>
`;
